api/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM alpine:3.15.0
```

Comment:
@Hareet I remember you saying that it would be better to use `node:alpine` ?

Reply:
I think it's good that we lock down the version because that always gets the latest version and the package libraries we want to get for the downstream packages like node may not exist for that version.